bin/load-test-db

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# no more tables
```

Comment:
Just put a comment that explains that this wipes all the tables in the test db and recreates them empty